pages/common/if.md

```diff
@@ -1,28 +1,36 @@
 # if
 
-> Simple shell conditional.
+> Performs conditional processing in shell scripts.
 > See also: `test`, `[`.
-> More information: <https://www.tcl.tk/man/tcl8.6.11/TclCmd/if.html>.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs>.
 
-- Execute two different commands based on a condition:
+- Execute the specified commands if the condition command exit status is zero:
 
-`if {{command}}; then {{echo "true"}}; else {{echo "false"}}; fi`
+`if {{condition_command}}; then {{commands}}; fi`
 
-- Check if a variable is defined:
+- Execute the specified commands if the condition command exit status is nonzero:
 
-`if [[ -n "{{$VARIABLE}}" ]]; then {{echo "defined"}}; else {{echo "not defined"}}; fi`
+`if ! {{condition_command}}; then {{commands}}; fi`
 
-- Check if a file exists:
+- Execute the first specified commands if the condition command exit status is zero otherwise execute the second specified commands:
```

Comment:
```suggestion
- Execute the first specified commands if the condition command's exit status is zero otherwise execute the second specified commands:
```